### ISSUE TITLE
How to change the ticket design 

### ISSUE THREAD

**Issue by rrmsz2** [mrf345/FQM#263]
Is there option to change the printed ticket design ?
adding logo and company name ?

**Comment by mrf345** [comment]
Yes, you can customize the printed ticket, try to change the header and sub-header:

![image](https://user-images.githubusercontent.com/26286907/198894247-f37a60c3-485f-4d6f-b6da-6213ce192625.png)

However, there's no support for adding a logo image, but shouldn't be too hard to implement. Happy to point out where to start and, what to-do in case someone else is interested in implementing this.
